tests/e2e/support.py

```diff
@@ -163,3 +163,31 @@ def create_kueue_resources(
             body=local_queue_json,
         )
         print(f"'{local_queue}' created in namespace '{self.namespace}'")
+
+
+def delete_kueue_resources(
+    self, cluster_queue="cluster-queue-mnist", flavor="default-flavor-mnist"
```

Comment:
Can you make those names global variables?
That way they will be kept in sync with creation functions.